python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/aaa-server-groups-ldap.j2

```diff
@@ -4,20 +4,23 @@
  that can be found in the LICENSE file.
 #}
 {# eos - aaa server groups ldap #}
-{% for aaa_server_group in aaa_server_groups | arista.avd.natural_sort('name') %}
-{%     if aaa_server_group.type == 'ldap' %}
+{% if aaa_server_groups is arista.avd.defined %}
+{%     set aaa_server_groups = aaa_server_groups | selectattr('type', 'arista.avd.defined') | list %}
```

Comment:
Instead of these just check for `aaa_server_group.type is arista.avd.defined('ldap')` inside for loop.
Same for radius and tacacs+

Reply:
Fixed.